modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsAutoIncrement.testcase

```diff
@@ -101,6 +101,99 @@ definition {
 		}
 	}
 
+	@description = "LPS-169223 - Verify after importing an object entry with a modified auto increment field value, new object entries increment based off of the modified value"
+	@priority = 4
+	test CanIncrementAfterImportingObjectEntry {
+		task ("Given an object definition called Shoes that contains an auto-increment field and a text field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Shoe",
+				objectName = "Shoe",
+				pluralLabelName = "Shoes");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Color",
+				fieldName = "color",
+				fieldType = "String",
+				isRequired = "true",
+				objectName = "Shoe");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "AutoIncrement",
+				fieldLabelName = "Identification",
+				fieldName = "identification",
+				fieldType = "String",
+				initialValue = 001,
+				isRequired = "false",
+				objectName = "Shoe",
+				prefix = "",
+				suffix = "-Sneakers");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "Shoe");
+		}
+
+		task ("and Given an object entry") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "color",
+				objectName = "Shoe",
+				value = "Blue");
+		}
+
+		task ("and Given the object entry is exported") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoExport();
+
+			ImportExport.configureExport(
+				entityType = "C_Shoe (v1_0 - Liferay Object REST)",
+				exportFileFormat = "JSON");
+
+			Button.click(button = "Export");
+
+			WaitForElementPresent(locator1 = "ImportExport#EXECUTION_SUCCESS");
+
+			ImportExport.unzipDownloadedExportFile();
+		}
+
+		task ("and Given the auto increment field value and the ERC are modified in the JSON file") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.gotoImport();
+
+			ImportExport.selectEntity(entityType = "C_Shoe (v1_0 - Liferay Object REST)");
+
+			UploadDependencyFile.uploadFile(fileName = "ImportedObjectEntriesShoes.json");
```

Comment:
Hey @andreefarias, we're trying to avoid using hardcoded object entries for our tests. The idea here is to modify the exported file and then import it.

Reply:
See https://liferay.atlassian.net/wiki/spaces/QA/pages/2157085195/Common+Poshi+Methods#StringUtil to modify strings.